src/Runner.Common/Logging.cs

```diff
@@ -24,6 +24,7 @@ public class PagingLogger : RunnerService, IPagingLogger
 
         private Guid _timelineId;
         private Guid _timelineRecordId;
+        private string _pageId;
```

Comment:
reverted from https://github.com/actions/runner/pull/255 to make sure page log name unique on disk.

Reply:
tested on my env and it works